.gitlab-ci.yml

```diff
@@ -0,0 +1,39 @@
+stages:
```

Comment:
This file is for gitlab only, and has been removed, please delete it.

Reply:
Done.